### DISCUSSION TITLE
the unit for recombination rate and methods to estimate global parameter confidence interval

### DISCUSSION THREAD

**Discussion by XieHongX** [LohseLab/gIMble#134] [Q&A] [answered]
Hi

Not sure if this should go to an issue or discussion.

When running gimble simulate, I want to use a fixed recombination. However, I am not sure shall I provide r =1.19 (cM/Mb), r = 1.19e-8 for the recombination rate per site per generation?
Then after obtaining the simulated data using global parameters, I then want to obtain the parametric bootstrap result for global parameters. Shall I use gimble optimize or gimble gridsearch using simulated data?

Thanks for any information.

**Comment by GertjanBisschop** [comment, marked as answer]
Hi @XieHongX , 
Thanks for reaching out. 
- You can run `gimble module --help` for each module, and this should provide clarity on most of the parameter questions you might have. In this case it is stated indeed that the recombination rate should be provided in the cM/Mb units.
- It is up to you how you want to find the best parameter combination for each simulation replicate. Using a grid makes a lot of sense to me given that this will dramatically reduce the computational cost because all model-based probabilities will have been precomputed when generating a grid.
